services/navigation/client.go

```diff
@@ -60,6 +60,8 @@ func (c *client) Mode(ctx context.Context, extra map[string]interface{}) (Mode,
 		return ModeManual, nil
 	case pb.Mode_MODE_WAYPOINT:
 		return ModeWaypoint, nil
+	case pb.Mode_MODE_EXPLORE:
```

Comment:
[note] No current client testing is done on SetMode Waypoint. The only test of SetMode is a single call to Manual, and so I have not included testing of Explore at this time.

If desired I can add a test, but I'd like to do it for both Waypoint and Explore

Current test:
```
// test set mode
extra = map[string]interface{}{"foo": "SetMode"}
err = workingNavClient.SetMode(context.Background(), navigation.ModeManual, extra)
test.That(t, err, test.ShouldBeNil)
test.That(t, receivedMode, test.ShouldEqual, navigation.ModeManual)
test.That(t, extraOptions, test.ShouldResemble, extra)
```

Reply:
yeah, this would be helpful

Reply:
Done